workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/WorkflowViewStub.kt

```diff
@@ -114,8 +115,8 @@ class WorkflowViewStub @JvmOverloads constructor(
   }
 
   /**
-   * Sets the visibility of this stub as usual, and also that of [actual].
-   * Any new views created by [update] will be assigned this visibility.
+   * Sets the visibility of this stub as usual, and also that of [actual] if it exists.
+   * Any new views created by [update] will NOT be assigned this visibility.
```

Comment:
Here's a thought: what if we always propagate the visibility of `actual`

Reply:
Maybe combination of that and your `update` idea. Give `update` an optional `@IdRes visibility : Int?` argument that defaults to null. If it's not set, we copy from `actual` (remember that `actual` defaults to `this`).

Reply:
Thinking some more about it, setting the visibility through update might not be a great idea since the order of inflation/visibility setting is ambiguous to the reader. It's easy enough to explicitly set the visibility after update if it needs to be overriden.